File: BMotionServices/Parse/ResponseFuels.cs
using BMotionServices.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BMotionServices.Parse
{
    public class ResponseFuels
    {
        public string status { get; set; }
        public string message { get; set; }
        public List<Fuel> Data { get; set; }
    }
}

File: app/src/main/java/com/stratone/bmotion/response/ResponseFuels.java
package com.stratone.bmotion.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.stratone.bmotion.model.Fuel;

import java.util.List;

public class ResponseFuels {
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("Data")
    @Expose
    private List<Fuel> data = null;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Fuel> getData() {
        return data;
    }

    public void setData(List<Fuel> data) {
        this.data = data;
    }
}
